### PR TITLE
Turn off CONTRACT_ACTION tracing in mainnet, testnet and performance testing.

### PR DESCRIPTION
**Pull request by lukelee-sl**
Signed-off-by: lukelee-sl <luke.lee@swirldslabs.com>

**Description**:
Explicitly only turn off tracing of CONTRACT_STATE_CHANGE and CONTRACT_BYTECODE in testnet, mainnet and preprod (used by performance testing).  By omitting CONTRACT_ACTION, we implicitly turn it off for these environments.

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/configuration/preprod/bootstrap.properties [hunk 1]
```diff
@@ -2,4 +2,5 @@ bootstrap.throttleDefsJson.resource=throttles.json
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```
Comment (Neeharika-Sompalli) [inline hedera-node/configuration/preprod/bootstrap.properties:2]: We should change the existing one and remove the newly added one.

Comment (lukelee-sl) [reply hedera-node/configuration/preprod/bootstrap.properties]: Thanks for catching that!

File: hedera-node/configuration/preprod/bootstrap.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 bootstrap.throttleDefsJson.resource=throttles.json
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```
Comment (Nana-EC) [inline hedera-node/configuration/preprod/bootstrap.properties:2]: Does this affect previewnet?
If so we should leave this and just ensure in our perfnet tests we explicitly override this

**Review by Nana-EC — COMMENTED** [review]
Are we missing the default to ensure it's set on everywhere else?

File: hedera-node/configuration/preprod/bootstrap.properties (hunk 2 above)
Comment (Nana-EC) [reply hedera-node/configuration/preprod/bootstrap.properties:2]: COnfirmed this doesn't affect previewnet

**Review by Nana-EC — APPROVED** [review]
LG

**Review by shemnon — APPROVED** [review]

**Pull request merged**